### PR TITLE
feat: Verifications API

### LINKED ISSUE #129: Verifications API
Implement [Verifications](https://w3c-ccg.github.io/vc-verifier-http-api/#/internal/verifyCredential) API

### PR DESCRIPTION
**Pull request by rolsonquadras**
- Implement https://w3c-ccg.github.io/vc-verifier-http-api/#/internal/verifyCredential
- BDD test

closes #129 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bdd/pkg/issuer/issuer_steps.go [hunk 1]
```diff
@@ -249,6 +286,25 @@ func (e *Steps) sendCreateRequest(req []byte) (*docdid.Doc, error) {
 	return didDoc, nil
 }
 
+func checkDIDAvailable(did string) error {
```
Comment (fqutishat) [inline test/bdd/pkg/issuer/issuer_steps.go]: https://github.com/trustbloc/edge-service/blob/f10282efb00746eb3a639b63bc85fd45448ef77f/test/bdd/pkg/vc/vc_steps.go#L606

Comment (fqutishat) [reply test/bdd/pkg/issuer/issuer_steps.go]: duplicate code

Comment (rolsonquadras) [reply test/bdd/pkg/issuer/issuer_steps.go]: Thanks. I wasn't aware of that code piece.

Comment (rolsonquadras) [reply test/bdd/pkg/issuer/issuer_steps.go]: The function you mentioned is not exposed. I'll move this along with other common code used across BDD as part of #132 as this change is already close to ~500 lines.

File: pkg/restapi/vc/operation/operations.go [hunk 2]
```diff
@@ -866,6 +875,77 @@ func (o *Operation) generateKeypairHandler(rw http.ResponseWriter, req *http.Req
...
+	verificationReq := CredentialVerificationsRequest{}
+
+	err := json.NewDecoder(req.Body).Decode(&verificationReq)
+	if err != nil {
+		o.writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf(invalidRequestErrMsg+": %s", err.Error()))
+
+		return
+	}
+
+	// TODO What are the checks to perform if options are not passed ? For now, made options and checks mandatory.
```
Comment (troyronda) [inline pkg/restapi/vc/operation/operations.go]: Default for empty checks seems to be all checks.

> A default list is also expected to encapsulate all available checks

Comment (troyronda) [reply pkg/restapi/vc/operation/operations.go]: @rolsonquadras will update in followup PR.

Comment (rolsonquadras) [reply pkg/restapi/vc/operation/operations.go]: Created an issue https://github.com/trustbloc/edge-service/issues/133

**Pull request merged**